server/views/admin/licences/activity.pug

```diff
@@ -8,6 +8,10 @@ mixin detail(type, details)
         + notify(details)
     when 'RESET'
         | Licence has been reset
+    when 'COMPLETE_IN_HDC'
+        | Licence to be completed in HDC
+    when 'COMPLETE_IN_CVL'
+        | Licence to be completed in CVL
```

Comment:
Looks like this in the service: 
<img width="944" alt="Screenshot 2024-12-09 at 15 35 44" src="https://github.com/user-attachments/assets/0cc8d252-0eec-4e91-bada-313fb11297b2">

Reply:
meh.. could we call have the details be "licence to be created in.." 

(don't worry about the routes and page names etc...)